packages/api-explorer/__tests__/PathUrl.test.jsx

```diff
@@ -18,6 +18,7 @@ const props = {
   oas,
   oauth: false,
   onChange: () => {},
+  onGroupChange: () => {},
```

Comment:
What does onGroupChange do?

Reply:
That gets called when you toggle between auth groups in the auth modal. Maybe it should be renamed to `onAuthGroupChange`.